src/docs/assets/portal.js

```diff
@@ -4,16 +4,12 @@ import log from "../../lib/log";
 
 const host = (window && window.location && window.location.hostname) || '';
 const parts = host.split('.');
+const GLOBAL_VENDOR_LIST_DOMAIN = 'https://vendorlist.consensu.org/vendorlist.json';
 const COOKIE_DOMAIN = parts.length > 1 ? `;domain=.${parts.slice(-2).join('.')}` : '';
 const COOKIE_MAX_AGE = 33696000;
 const COOKIE_NAME = 'euconsent';
 
-const readVendorListPromise = fetch('./vendors.json', {
-	headers: {
-		'Accept': 'application/json',
-		'Content-Type': 'application/json'
-	}
-})
+const readVendorListPromise = fetch(GLOBAL_VENDOR_LIST_DOMAIN)
```

Comment:
shall we not keep the headers check ?

Reply:
The headers check was forcing an OPTIONS call which was wreaking havoc with the requests. I also saw that the AppNexus implementation had taken out those headers as well, so I went with getting rid of them.